ethicml/data/csvs/make_synthetic_4.py

```diff
@@ -0,0 +1,110 @@
+"""Generate Synthetic data."""
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from ethicml.data.csvs.utility import sigmoid
+
+
+def main() -> None:
+    r"""Make synthetic data.
+
+    Generate synthetic data that conforms to 'Scenario 4'
+
+    Scenario 4
+    ----------
+                           +---+
+                 +---------+ S |
+                 |         +-+-+
+                 v           |
+    +-----+   +--+--+        |
+    | Y_1 +<--+ X_1 +-+      v
+    +-----+   +-----+ |   +--+--+
+                      +-->+ Y_3 |
+    +-----+   +-----+ |   +-----+
+    | Y_2 +<--+ X_2 +-+
+    +-----+   +-----+
+
+    In this scenario, there are two input variables, X_1 and X_2.
+    There are three outcome variables, Y_1, Y_2 & Y_3.
+    There is one sensitive attribute, S, which is independent of X_2 & Y_2,
+    but not independent of X_1, Y_1, or Y_3.
+
+    We have:
+    S ~ B(0.5)
+
+    X_1 ~ N(2*S, 2)
+    X_2 ~ N(-1.5, 4)
+
+    Y_1 ~ B(sigmoid(X_1)))
+    Y_2 ~ B(sigmoid(X_2))
+    Y_3 ~ B(sigmoid(((X_1 + 2*X_2)/2)+S))
```

Comment:
btw, you can also use latex math in sphinx docs

Reply:
cool - should probably make use of that. The scripts to make the data aren't included in the docs though, so it will make it quite cluttered in this case